pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/SimpleActivityRevisionTask.java

```diff
@@ -0,0 +1,335 @@
+
+        this.studyDto = handle.attach(JdbiUmbrellaStudy.class).findByStudyGuid(dataCfg.getString("study.guid"));
+
+        List<? extends Config> activityUpdateConfigs = dataCfg.getConfigList(STUDY_UPDATES);
+
+        this.sqlHelper = handle.attach(SimpleActivityRevisionTask.SqlHelper.class);
+        this.activityDao = handle.attach(ActivityDao.class);
+        this.jdbiVarSubst = handle.attach(JdbiVariableSubstitution.class);
+        this.jdbiRevision = handle.attach(JdbiRevision.class);
+        this.templateDao = handle.attach(TemplateDao.class);
+        this.jdbiBlockContent = handle.attach(JdbiBlockContent.class);
+
+        activitiesUpdate(handle, activityUpdateConfigs);
+    }
+
+    private ActivityVersionDto getVersion(Handle handle, RevisionMetadata meta, String activityCode, String versionTag) {
+        long activityId = ActivityBuilder.findActivityId(handle, studyDto.getId(), activityCode);
+        return activityDao.changeVersion(activityId, versionTag, meta);
+    }
+
+    private void activitiesUpdate(Handle handle, List<? extends Config> activityConfigList) {
+        for (Config activityConfig: activityConfigList) {
+            String activityCode = activityConfig.getString("activityCode");
+            String versionTag = activityConfig.getString("versionTag");
+
+            log.info("Changing version of {} to {} with timestamp={}", activityCode, versionTag, timestamp);
+            long ts = this.timestamp.toEpochMilli();
+
+            String reason = String.format(
+                    "Update activity with studyGuid=%s activityCode=%s to versionTag=%s",
+                    studyDto.getGuid(), activityCode, versionTag);
+            RevisionMetadata metaConsent = new RevisionMetadata(ts, adminUser.getId(), reason);
+
+            ActivityVersionDto activityVersion = getVersion(handle, metaConsent, activityCode, versionTag);
+            activityUpdate(metaConsent, activityVersion, activityConfig);
+        }
+    }
+
+    private void activityUpdate(RevisionMetadata meta, ActivityVersionDto version, Config activityConfig) {
+        updateTemplates(meta, version, activityConfig);
+        updateTemplateVariables(meta, version, activityConfig); //revision only variables in the Template
+        updateQuestionTemplateVariables(meta, version, activityConfig); //revision only variables in the Question Template
+    }
+
+    private void updateTemplates(RevisionMetadata meta, ActivityVersionDto version, Config activityConfig) {
+        log.info("UPDATE Templates/Content Blocks");
+        List<? extends Config> configList = null;
+
+        try {
+            configList = activityConfig.getConfigList(BLOCK_UPDATES);
+        } catch (Exception e) {
+            log.info("No Template updates found for activity={}", version.getActivityId());
+            return;
+        }
+
+        for (Config config : configList) {
+            revisionContentBlockTemplate(meta, version, config);
+        }
```

Comment:
Do you want to add another `log.info` at the end of this function to indicate operation has completed? Like in `updateQuestionTemplateVariables()`.